modules/acquia_cms_headless/acquia_cms_headless.install

```diff
@@ -33,3 +39,49 @@ function acquia_cms_headless_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Update all consumers and assign content model roles and add
+ * provide `issue subrequests` permission to authenticated & anonymous role.
+ */
+function acquia_cms_headless_update_8001() {
+  $rolesStorage = \Drupal::entityTypeManager()->getStorage('user_role');
+  $query = $rolesStorage->getQuery();
+  $roleIds = $query
+    ->condition('id', ["authenticated", "anonymous", "headless"], 'NOT IN')
+    ->execute();
+
+  $consumerStorage = \Drupal::entityTypeManager()->getStorage('consumer');
+  $query = $consumerStorage->getQuery();
+  $consumerIds = $query
+    ->exists('user_id')
+    ->execute();
+  $consumers = $consumerStorage->loadMultiple($consumerIds);
+  foreach ($consumers as $consumer) {
```

Comment:
Are we sure we want to do this was all consumer users because sites could have consumers that are used for other purposes? I think we should do this only for the default consumer created by ACMS.

Reply:
Ok, will update this.